frontend/src/libs/components/toastr/toastr.tsx

```diff
@@ -0,0 +1,10 @@
+import 'react-toastify/dist/ReactToastify.css';
+
+import React from 'react';
+import { ToastContainer } from 'react-toastify';
+
+const Toastr: React.FC = () => {
```

Comment:
Let's call this wrapper component `Notification`
+ don't forget to rename file\folder